packages/create-hint/src/templates/locales-messages.json.hbs

```diff
@@ -0,0 +1,16 @@
+{
+    {{#each hints}}
+    "{{#if ../isMulti}}{{toCamelCase normalizedName}}_{{/if}}description": {
+        "description": "Metadata description",
+        "message": "{{description}}"
+    },
+    "{{#if ../isMulti}}{{toCamelCase normalizedName}}_{{/if}}name": {
+        "description": "Metadata name",
+        "message": "{{normalizedName}}"
+    },
+    "{{#if ../isMulti}}{{toCamelCase normalizedName}}_{{/if}}reportMessage": {
```

Comment:
Other than `name` and `description`, we don't generally prefix messages in `messages.json`, even for multi-hints. The name and description metadata is an exception because we need those names to follow a well-defined format so we can use them when generating documentation.
```suggestion
    "reportMessage": {
```

Reply:
@antross  that is right if user create new messages in locales file himself. But here I am auto-generating, and for demo purpose, I am keeping name of key as `reportMessage`. Now what should I do for multiple hints here? Key name has to be prefixed with something.

Reply:
Or should I just add only one message as `reportMessage` and use it in all hints of multi hint package

Reply:
Good point. I like your suggestion of using a common message in this case. Otherwise I'm concerned new users would think they have to prefix all messages.

Reply:
Thanks @antross. And I have made the change. Please review last commit.